### DISCUSSION TITLE
Why would the Google lighthouse give me a bad ratio from a picture?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#64310] [Help] [answered]
### Summary

This error:
![image](https://github.com/vercel/next.js/assets/93617433/8e94efc0-4321-46e4-9966-fc10c471cdd9)

^^ This picture above is indeed 700x400 pixels.

The code is this:
```
<span>
            <Image
              src="/images/maintenance-1.png"
              id="maintenance1"
              alt="maintenance lubrication"
              width={700}
              height={400}
              priority={true} // {false} | {true}
            />
          </span>
```

Why would the Lighthouse be saying I got the wrong aspect ratio showing up?

**Comment by icyJoseph** [comment, marked as answer]
Try giving the image a CSS property of `height: auto;`

**Reply by Elindo586** [reply]
Thanks dude... you are awesome!

![image](https://github.com/vercel/next.js/assets/93617433/7227a689-c1b4-4ab1-8aee-78f884116fb4)

**Reply by Elindo586** [reply]
Now I just gotta make sure all pages are 100%... how fun..
